release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -86,28 +106,24 @@ These steps take approximately one hour to complete, and should be completed imm
     - [kubernetes-release-team-shadows](https://groups.google.com/a/kubernetes.io/g/release-team-shadows)
         - ⚠️  List of members for this group is managed in git. Please create a PR against `kuberenetes/sig-release` repo to include your email under `release-team-shadows` group in [`sig-release/group.yaml`](https://github.com/kubernetes/k8s.io/blob/main/groups/sig-release/groups.yaml)
 
-- Make sure you're included in the "official" release team file, e.g: [release 1.21](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.21/release-team.md). If not submit a PR and add yourself.
```

Comment:
It looks like you may have deleted this section. Was that intentional? If not, it looks like it should be re-added.

Reply:
yes! It's duplicated. it's in the list `⚠️ Tasks to DO`